bin/deliver_poste_messages

```diff
@@ -270,18 +270,26 @@ def main()
     until queue.empty?
       next unless delivery = queue.pop(true) rescue nil
 
+      sent_at = DateTime.now
+
       begin
         deliverer.send delivery
-      rescue Net::SMTPSyntaxError => e
-        puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
-      rescue Net::SMTPFatalError => e
+      rescue Net::SMTPSyntaxError, Net::SMTPFatalError => e
         puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
+      rescue RuntimeError => e
+        if e.message.to_s.start_with? 'ABORT:'
```

Comment:
I agree that would be much cleaner, but then the ERB in the email template .md files would have to know about that class. I was trying to keep it simple from that standpoint.

Now that I think about it, the calling context should already know about this class if it's required there? I'll experiment...